mvpa2/algorithms/hyperalignment.py

```diff
@@ -198,7 +198,7 @@ def train(self, datasets):
             ref_ds = np.argmax(nfeatures)
         else:
             ref_ds = params.ref_ds
-            if ref_ds < 0 and ref_ds >= ndatasets:
+            if ref_ds >= ndatasets:
```

Comment:
Ok, very good, thanks. How about adding a comment indicating that the Parameter class takes care of this? It may also be good to add two other cases to the test_hyper_ref_ds_range_checks function, using negative and non-integer ref_ds values.